shuup/core/suppliers/base.py

```diff
@@ -24,6 +25,7 @@ class BaseSupplierModule(object):
 
     identifier = None
     name = None
+    handels_internal_type = None
```

Comment:
why do you need this? this is not even needed

Reply:
Yes as this is the base class for all the supplier modules.